Tiltfile.dev

```diff
@@ -5,7 +5,7 @@ local_resource(
 )
 
 custom_build(
-   ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/lfp-pay-web',
+   ref = '169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/lfp-pay-web',
```

Comment:
Just to confirm CH are moving away from using Tilt (deprecated) locally with preference for just `chs-dev` deployments. 

Mainly wanted to say that the local image for lfp-pay-web is actually still on eu-west-1 development currently which is confusing.

See FYI https://eu-west-2.console.aws.amazon.com/ecr/repositories/private/169942020521/local/lfp-pay-web?region=eu-west-2